### PR TITLE
feat: DEV-1602: Update Table Tag to show csv data

### PR DESCRIPTION
**Pull request by superdev808**

### CODE REVIEW

**Review by hlomzik — CHANGES_REQUESTED** [review]
Also don't update lock-files unless you change `package.json`

File: src/env/development.js [hunk 1]
```diff
@@ -55,6 +55,7 @@ import { RichTextPlainRemote } from "../examples/rich_text_plain_remote";
 import { Pairwise } from "../examples/pairwise";
 import { Repeater } from "../examples/repeater";
 import { Table } from "../examples/table";
+import { TableCsv } from "../examples/table_csv";
```
[the remark this answers is not in the document]
Comment (superdev808) [reply src/env/development.js:58]: not sure what this comment means.

File: src/tags/object/Table.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+          self._value = typeof originData === "string" ? JSON.parse(originData) : originData;
+          break;
+      }
+    }),
+  }));
+
+const TableModel = types.compose("TableModel", Base, ProcessAttrsMixin, AnnotationMixin, Model);
+
+const HtxTable = inject("store")(
+  observer(({ _, item }) => {
```
Comment (hlomzik) [inline src/tags/object/Table.js:107]: this is object, so `_` is redundant

File: src/tags/object/Table.js [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+        return Object.keys(self._value[0]).map(value => ({ title: value, dataIndex: value }));
+      }
+    },
+  }))
+  .actions(self => ({
+    updateValue: flow(function*(store) {
+      const { type, options } = parseTypeAndOption(self.valuetype);
+      let originData = parseValue(self.value, store.task.dataObj);
+
+      console.log(originData);
```
Comment (hlomzik) [inline src/tags/object/Table.js:67]: the last one left :)

**Review by hlomzik — APPROVED** [review]

**Comment by hlomzik** [comment]
and resolve conflict please, any version is good, they are the same

**Review by nicholasrq — APPROVED** [review]

**Pull request merged**